Vagrantfile

```diff
@@ -80,6 +80,13 @@ $provision_script = <<SCRIPT
 set -x
 set -e
 set -o pipefail
+# If the host is running SELinux remount the /sys/fs/selinux directory as read only,
+# needed for apt-get to work.
+if [ -d "/sys/fs/selinux" ]; then
+  sudo mount -o remount,ro /sys/fs/selinux
+fi
+# Set default locale, this prevents errors on vagrant ssh
+echo "LC_ALL=en_US.UTF-8" | sudo tee -a /etc/default/locale
```

Comment:
@andersk given that context, do you still think we shouldn't be setting LC_ALL in the Vagrant guests?  It's a controlled environment, and we do want commands to run with a fixed locale for predictability, so to me it seems like a potentially reasonable solution...